atd-etl/cris_import/cris_import.py

```diff
@@ -51,6 +53,14 @@ def main(cli_args):
             csv_records_processed_dict = process_csvs(extract_dir)
             records_processed.update(csv_records_processed_dict)
 
+            # update the import log to capture # of csvs processed
+            update_log_entry(
+                log_entry_id=log_entry_id,
+                payload={
+                    "records_processed": records_processed,
+                },
+            )
```

Comment:
i added this so that the number of CSVs processed will be saved in the import log even if the PDF import fails. the `records_processed` will look something like this:

```
{"pdfs": 0, "units": 82, "charges": 16, "crashes": 38, "persons": 85}
```

and the `completed_at` value will be null, indicating that the import is not complete.